frontend/src/assets/i18n/en.json

```diff
@@ -377,6 +377,7 @@
             "show-result": "Show result in map view",
             "hide-result": "Hide result in map view",
             "visible-result": "Result is currently visible in map view",
+            "purged-result": "Data for this result has been purged\nBecause the result will need to be recalculated, it takes a little longer to load.",
```

Comment:
For clarity I would recommend using a slightly different wording for the second sentence like:
 - "Because the result will need to be recalculated, it **will take** a little longer to load." OR
- "This will take a little longer to load because the result is being recalculated." OR 
- "The result is being recalculated. This will take a little longer to load.